packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -224,23 +250,75 @@ abstract class RenderSliverMultiBoxAdaptor extends RenderSliver
 
   bool _debugAssertChildListLocked() => childManager.debugAssertChildListLocked();
 
+  /// Verify the child list index is in strictly increasing order.
+  ///
+  /// This is a no-op for non-debug builds.
+  void _debugVerifyChildOrder(){
+    assert(() {
+      if (_debugChildIntegrityEnabled) {
+        if (firstChild != null) {
```

Comment:
right now this will be run when performRebuild. Need to make sure when the this is empty it still pass